src/components/Badge/Badge.st.css

```diff
@@ -54,6 +54,11 @@
     width: 2px;
 }
 
+.icon::before {
+    content: '';
+    margin: 0 -1px;
+}
+
```

Comment:
🤔  I looked at the storybook, but it doesn't seem to work
Also, if for instance we're in `ltr`, the `margin-right: -1px` won't move the icon left, but move what's after the icon to the left.

I think we should just replace the `root`'s padding with a `before` and `after` instead.

Reply:
Are you sure? this is why I did it on both sides